GestioneOrari/src/Elaborato/Elaborato.java

```diff
@@ -1,8 +1,23 @@
 package Elaborato;
 
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import row.Row;
+
 public class Elaborato {
```

Comment:
So che l'ho fatto io, ma riesci a trovare un nome in inglese? Manteniamo la convenzione inglese.

Pensando che è un dato elaborato si immagina che sia un risultato di un procedimento quindi un dato processato, forse Processed o qualcosa di simile come nome della classe sarebbe meglio.